### PR TITLE
Major cleanup of codebase

### PR DESCRIPTION
**Pull request by bobveringa**
Significantly cleaned up the code:
* Added helper functions for common constructions.
* Added docstrings to most functions
* Added (some) comments
* Rewrote canvas id system to allow easier expansion (or reduction)
* Removed a bunch of magic numbers

### CODE REVIEW

**Comment by bobveringa** [comment]
Updates to resolve conflicts.

File: placenlbot.user.js [hunk 1]
```diff
@@ -30,6 +30,19 @@ var currentPlaceCanvas = document.createElement('canvas');
 // Global constants
 const DEFAULT_TOAST_DURATION_MS = 10000;
 
+const R_PLACE_WIDTH = 2000;
+const R_PLACE_HEIGHT = 2000;
+
+const SCRIPT_BRAND = 'userScriptV21'
```
Comment (NoahvdAa) [inline placenlbot.user.js]:
```suggestion
const SCRIPT_BRAND = 'userScriptV26';
```

Comment (NoahvdAa) [reply placenlbot.user.js]: Actually, that should be 27 (version st the top needs to be updated too)

**Comment by bobveringa** [comment]
Everything seems to be running fine.

**Pull request closed without merging**